rolepermissions/utils.py

```diff
@@ -24,3 +24,10 @@ def camelToSnake(s):
 
     subbed = _underscorer1.sub(r'\1_\2', s)
     return _underscorer2.sub(r'\1_\2', subbed).lower()
+
+
+def snake_to_title(s) :
+    return ' '.join(x.capitalize() for x in s.split('_'))
+
+def camel_or_snake_to_title(s):
```

Comment:
Can you provide some tests for those functions?

Reply:
Yes.  Good catch.